modules/ROOT/pages/intro-error-handlers.adoc

```diff
@@ -84,7 +84,8 @@ For the most part, Mule 3 only allows error handling at the flow level, forcing
 
 image::error-handling-try.png[Try Scope]
 
-The error handler behaves as we have explained earlier. In the example above, any database connection errors are propagated, causing the `try` to fail and the flow’s error handler to execute. In this case, any other errors are handled, and the Try scope is considered successful which, in turn, means that the next processor in the flow, an HTTP request, continues its execution.
+In the above example, On Error Propagate component is configured to propagate database connection errors, causing the `try` to fail and the flow's error handler to execute.
```

Comment:
According to our guidelines: Because documentation is presented on many different types of devices with various amounts of visible space, replace references to text or graphical elements that are "above" or "below" in the narrative flow with "previous" and "following":

Suggestion: In the previous example